content/nav-openmaterial.adoc

```diff
@@ -25,7 +25,7 @@
 ** xref:geometry/asset-schema.adoc[]
 ** xref:geometry/mapping-schema.adoc[]
 * xref:material/material-index.adoc[]
-** xref:material/introduction.adoc[]
```

Comment:
The wrong line is removed here. Unfortunately I just spotted this after I merged. I will create a new PR to fix that.

Reply:
Just saw it myself, sorry for that, I replaced the wrong line.